### PR TITLE
Release-4.0.0 HDS-2424 fix mobile title sizes in Hero

### PR DESCRIPTION
**Pull request by mrTuomoK**
## Description

Fixes mobile Hero font-size

## Related Issue

Closes [HDS-2424](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2424)

## How Has This Been Tested?

- running tests

## Demos:

Links to demos are in the comments

## Add to changelog

- [X] Added needed line to changelog

[HDS-2424]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/components/hero/hero.css [hunk 1]
```diff
@@ -278,6 +278,10 @@
     padding-right: var(--spacing-m);
     padding-top: var(--spacing-xs);
   }
+
+  .hds-hero__title {
+    font-size: var(--fontsize-heading-xl);
```
Comment (SillanmakiA) [inline packages/core/src/components/hero/hero.css:283]: Specifications were not the clearest. Use --fontsize-heading-xl-mobile also in this breakpoint between 768px and 992px.

**Review by SillanmakiA — CHANGES_REQUESTED** [review]
Use --fontsize-heading-xl-mobile also in "tablet" breakpoint 768-991px.

**Comment by mrTuomoK** [comment]
Fixed the issue @SillanmakiA 👍 Although it's not caught by our tests since we only check mobile and desktop sizes but can be tested in the demos ofc.

**Review by SillanmakiA — APPROVED** [review]
It's perfect. 👌🏻

**Pull request merged**